### PR TITLE
Add a project that copies perf scripts to build output.

### PR DESCRIPTION
**Pull request by rchande**
cc @TyOverby

### CODE REVIEW

File: src/Test/Perf/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
```
Comment (TyOverby) [inline src/Test/Perf/Class1.cs]: Remove this?

**Comment by rchande** [comment]
@TyOverby done

File: src/Test/PerformanceTesting.csproj [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    </Content>
+    <Content Include="Perf\util\test_util.csx">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Perf\util\tools_util.csx">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Perf\util\trace_manager_util.csx">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
```
Comment (TyOverby) [inline src/Test/PerformanceTesting.csproj]: Should we have one of these for `HelloWorld.cs`

**Comment by TyOverby** [comment]
Is there going to be an equivalent change in your roslyn-internal PR?

File: src/Test/PerformanceTesting.csproj (hunk 2 above)
Comment (rchande) [inline src/Test/PerformanceTesting.csproj]: Shouldn't we? Isn't it consumed by helloworld.csx?

Comment (TyOverby) [inline src/Test/PerformanceTesting.csproj]: Oh, I see it down in the other ItemGroup.  I thought you were missing it.

What are these itemgroups anyway?

**Comment by TyOverby** [comment]
LGTM

File: src/Test/PerformanceTesting.csproj (hunk 2 above)
Comment (rchande) [inline src/Test/PerformanceTesting.csproj]: They are whatever the project system generated :smile:

**Comment by TyOverby** [comment]
Should this project be added to any of the solutions?

**Pull request closed without merging**